ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -635,6 +635,21 @@ tests:
     env:
       BASE_DOMAIN: alicloud-dev.devcluster.openshift.com
     workflow: openshift-e2e-alibabacloud-disk-csi
+- as: release-analysis-install-analysis
```

Comment:
suggest release-analysis-install-success just to not double analyze. would this break consistency with anything else?

or release-payload-install-analysis.

what will happen when this job runs once a year? :) failure I assume as it has no payload data to examine. Will be an interesting one to find in sippy.

Reply:
yees, I'm expecting failure but only once a year -- and hoping that's just noise.